index.html

```diff
@@ -7,12 +7,59 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="style.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+  <nav class="navigation"> 
+    <div>
+      <ul class= "links">
+        <li class="current-page"><a href="">Meet Karma</a></li>
+        <li><a href="">How It Works</a><li>
+        <li><a href="">Store</a></li>
+        <li><a href="">Blog</a></li>
+        <li><a href="">Help</a></li>
+        <li><a href="">Login</a></li>
+      </ul>
+      <img class="logo" src="img/karma-logo.svg" alt="Logo Image" width="26px" height="70px">
+    </div>
+  </nav>
+  <section class="introduction">
+    <div class="content"> </div>
+      <h1> Introducing Karma</h1>
+      <h3> Bring WiFi with you, everywhere you go.</h3>
+    <div class ="buttons">
+     <button> Learn More </button>
+    </div>
+  </section>
+  <section class="cases">
+    <div class="content">
+       <h2>Everyone needs a little Karma.</h2>
+       <ul class="devices">
+         <li><img src="img/icon-devices.svg" width="127px">
+         <div class="caption">
+          <h3> Internet for all devices</h3>
+         </div>
+         </li>
 
+         <li><img src="img/icon-coffee.svg" width="127px">
+          <div class="caption">
+            <h3> Boost your productivity</h3>
+          </div>
+          </li>
+          
+          <li><img src="img/icon-refill.svg" width="127px">
+            <div class="caption">
+              <h3> Pay as you go</h3>
+            </div>
+            </li>
+          </ul>
+          
+    </div>
+
+  </section>
```

Comment:
Something to finish the website would be to add a footer with the social media links - have a look at how the karma website is laid out and consider what style properties could be applied to the class to ensure that they are centered and evenly spaced.